app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
+  extend Devise::Models
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+  include DeviseTokenAuth::Concerns::User
+  validates_presence_of :role, :company_name, :company_url
```

Comment:
we can't really validate presence of company_name and company_url since develuper will not have them.

Reply:
but we cannot not have them since client will need them